api/testutils/kusttest/harnessenhanced.go

```diff
@@ -166,6 +166,24 @@ func (th *HarnessEnhanced) LoadAndRunGenerator(
 	return rm
 }
 
+func (th *HarnessEnhanced) LoadAndRunGeneratorWithBuildAnnotations(
```

Comment:
Any reason not to have `LoadAndRunGenerator` just call this one and then remove the annotations?

Reply:
good idea

Reply:
done